deploy/clowdapp.yml

```diff
@@ -0,0 +1,109 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: insights-storage-broker
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: insights-storage-broker
+  spec:
+    envName: ${ENV_NAME}
+    pods:
+    - name: service
+      image: ${IMAGE_NAME}:${IMAGE_TAG}
+      minReplicas: 1
+      web: true
+      livenessProbe:
+        failureThreshold: 3
+        httpGet:
+          path: /metrics
+          port: 9000
+          scheme: HTTP
+        initialDelaySeconds: 10
+        periodSeconds: 10
+        successThreshold: 1
+        timeoutSeconds: 27
+      readinessProbe:
+        failureThreshold: 3
+        httpGet:
+          path: /metrics
+          port: 9000
+          scheme: HTTP
+        initialDelaySeconds: 10
+        periodSeconds: 10
+        successThreshold: 1
+        timeoutSeconds: 30
+      resources:
+        limits:
+          cpu: ${CPU_LIMIT}
+          memory: ${MEMORY_LIMIT}
+        requests:
+          cpu: 250m
+          memory: 64Mi
+        volumeMounts:
+        - mountPath: /var
+          name: storage-broker-map
+        volumes:
+        - configMap:
+            defaultMode: 420
+            name: storage-broker-map
+          name: storage-broker-map
+      env:
+        - name: CLOWDER_ENABLED
+          value: ${CLOWDER_ENABLED}
+        - name: LOG_LEVEL
+          value: ${LOG_LEVEL}
+    objectStore:
+      - insights-dev-upload-perm
```

Comment:
these are the dev buckets, but the prod buckets would need to be used in production. Not sure if that needs to be reflected here.